backend/__init__.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
+
+
+# Put app and db here so the entire app could import them.
+app = Flask(__name__)
+
+if app.config["ENV"] == "production":
+    app.config.from_object("backend.config.ProductionConfig")
+else:
+    app.config.from_object("backend.config.DevelopmentConfig")
+    
```

Comment:
This isn't bad, just a suggestion for the future, when doing something similar:
a cool way of doing this would be to have a dictionary with the various configurations in the backend/config file:
config_dict = {"production: "backend.config.ProductionConfig", "devel": "backend.config.DevelopmentConfig"}
and then you could use:
app.config.from_object(config_dict[app.config["ENV"]]).

Then if you wanted to be super awesome, you could extend that, so you have a class variable in each subclass, "env_keyword" (or similar), and build the dictionary dynamically, so if you add new Config subclasses they will automatically be usable.

Reply:
Ack.